tests/integration.rs

```diff
@@ -792,7 +792,7 @@ mod integration_tests {
     async fn wait_tcp(url: &str, process: &mut Child, target: &str) -> Result<()> {
         let mut wait_count = 0;
         loop {
-            if wait_count >= 180 {
+            if wait_count >= 240 {
```

Comment:
Why are we bumping this?

Reply:
If you check failed workflow, most of the time it's because some applications take too much time to start. So I tried to bump the waiting time for it. For example:
* https://github.com/fermyon/spin/runs/7878118080?check_suite_focus=true
* https://github.com/fermyon/spin/runs/7900079763?check_suite_focus=true

```
...
connect 127.0.0.1:49288 error Connection refused (os error 61), retry 177
connect 127.0.0.1:49288 error Connection refused (os error 61), retry 178
connect 127.0.0.1:49288 error Connection refused (os error 61), retry 179
thread 'integration_tests::e2e_tests::test_bindle_static_assets' panicked at 'Ran out of retries waiting for ./target/debug/spin to start on URL 127.0.0.1:49288', tests/integration.rs:796:17
```